src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js

```diff
@@ -169,10 +169,13 @@
     $(".page").hide();
     $("#stackpage").show();
 
-    $("#rawExceptionButton").click(function () {
+    $("#rawExceptionButton").click(function (event) {
```

Comment:
If this was vanilla JS `event.preventDefault()` would have been the right thing to do. It's ok to return false in a jquery handled event: http://www.mail-archive.com/jquery-en@googlegroups.com/msg71371.html